### PR TITLE
Couchbase - GetAutoFailOverSettings - action && unit test.

### PR DESCRIPTION
**Pull request by mihaitusa**
Signed off by Tusa Mihai: tusa.mihai@gmail.com

### CODE REVIEW

**Comment by mihaitusa** [comment]
Tested -> functional!

File: cs-couchbase/src/main/java/io/cloudslang/content/couchbase/actions/cluster/GetAutoFailOverSettings.java [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+     *                             of the subject-alts. A wildcard can occur in the CN, and in any of the subject-alts.
+     *                             The only difference between "browser_compatible" and "strict" is that a wildcard (such
+     *                             as "*.foo.com") with "browser_compatible" matches all subdomains, including "a.b.foo.com".
+     *                             Valid values: "strict", "browser_compatible", "allow_all"
+     *                             Default value: "allow_all"
+     * @param trustKeystore        Optional - pathname of the Java TrustStore file. This contains certificates from other
+     *                             parties that you expect to communicate with, or from Certificate Authorities that you
+     *                             trust to identify other parties. If the protocol (specified by the "url") is not "https"
+     *                             or if trustAllRoots is "true" this input is ignored.
+     *                             Default value: ../java/lib/security/cacerts
```
Comment (lucian-cm) [inline cs-couchbase/src/main/java/io/cloudslang/content/couchbase/actions/cluster/GetAutoFailOverSettings.java:91]: Not sure about this default value: isn't it taken from system as all workers will have -Djavax.net.ssl.trustStore property set during initilization.

Comment (mihaitusa) [reply cs-couchbase/src/main/java/io/cloudslang/content/couchbase/actions/cluster/GetAutoFailOverSettings.java:91]: Ran it against a secured cluster (https, with a certificate to authenticate it) added that certificate here: <java_jdk_instalation_folder>\jre\lib\security -> works as expected.

Comment (mihaitusa) [reply cs-couchbase/src/main/java/io/cloudslang/content/couchbase/actions/cluster/GetAutoFailOverSettings.java:91]: Of course, those default values means least secured call...

**Comment by mihaitusa** [comment]
@lrevnic It's up to you!

File: cs-couchbase/src/main/java/io/cloudslang/content/couchbase/actions/cluster/GetAutoFailOverSettings.java (hunk 1 above)
Comment (lucian-cm) [reply cs-couchbase/src/main/java/io/cloudslang/content/couchbase/actions/cluster/GetAutoFailOverSettings.java:91]: Thanks for the information @mihaitusa, I marked the comment as resolved.

**Pull request merged**